media/js/mozorg/contact-spaces.js

```diff
@@ -160,6 +160,7 @@
                     // Hide spaces and community nav+meta
                     mozMap.toggleNav('contact');
                     mozMap.toggleMobileNav('contact');
+                    mozMap.setMapState();
```

Comment:
This still doesn't fix this issue you describe when I test locally (switch from a specific space, for example, back to contact us).